src/Stratis.Bitcoin/Consensus/ChainedHeaderTree.cs

```diff
@@ -52,6 +50,16 @@ public interface IChainedHeaderValidator
     /// </remarks>
     public interface IChainedHeaderTree
     {
+        /// <summary>
+        /// Gets size of unconsumed blocks data in bytes.
```

Comment:
It is not method, so probably it does not "get" the size but it is the size.

Reply:
Fixed